release-process.md

```diff
@@ -319,4 +319,17 @@ Once everything is working (website docs, website changes) create an announcemen
 and then send an e-mail to the mailing list. To create an announcement, create a post under
 `news/_posts` and then run `jekyll build`.
 
+<h4>Update a test suite depending on old releases</h4>
+
+Since Spark always keeps the latest maintenance release of each branch in the mirror network,
+we need to update a test suite to check backward compatibility by testing against old versions.
```

Comment:
Just say `HiveExternalCatalogVersionsSuite` here too I think and remove the next sentence.